Session Replay for mobile product docs (#10628)
getsentry/sentry-docs: docs/product/explore/session-replay/web/index.mdx

---
title: "Session Replay for Web"
sidebar_order: 10

> description: "Use Session Replay to get video-like reproductions of user interactions to improve your website or app experience."

Reviewer: ```suggestion
description: "Use Session Replay to get video-like reproductions of user interactions to improve your website experience."
```
since this one is web specific

![Session Replay User Interface](./img/session-replay-what-is-a-replay.png)

The start of a session replay recording can be triggered by:

- A user session being part of a sampling rate, as controlled by [`replaysSessionSampleRate`](/platforms/javascript/session-replay/#sampling). (When a user loads a page, a decision is made whether to sample it or not.)
- An error occurring during a session that’s not being recorded. The session is then recorded based on [`replaysOnErrorSampleRate`](/platforms/javascript/session-replay/#sampling).
- Manually calling the [`replay.start()`](/platforms/javascript/session-replay/understanding-sessions/#manually-starting-replay) method.

The end of a session replay recording can be triggered by:

> - User inactivity within the tab or page that’s being recorded. (It's considered inactivity when a user doesn’t click or navigate around the site for more than 15 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)

Reviewer 2: ```suggestion
- User inactivity within the tab or page that’s being recorded. (It's considered inactivity when a user doesn’t click or navigate around the site for more than 15 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)
- User closes the tab or page that's being recorded.
```

> - User inactivity within the tab or page that’s being recorded. (It's considered inactivity when a user doesn’t click or navigate around the site for more than 15 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)

Reviewer 2: @michellewzhang good catch! added :)

---
title: "Session Replay for Web"
sidebar_order: 10

> description: "Use Session Replay to get video-like reproductions of user interactions to improve your website experience."

Reviewer 3: ```suggestion
description: "Learn how the Session Replay feature can get video-like reproductions of user interactions so you can create a better web experience for your users."
```

![Session Replay User Interface](./img/session-replay-what-is-a-replay.png)

The start of a session replay recording can be triggered by:

- A user session being part of a sampling rate, as controlled by [`replaysSessionSampleRate`](/platforms/javascript/session-replay/#sampling). (When a user loads a page, a decision is made whether to sample it or not.)
- An error occurring during a session that’s not being recorded. The session is then recorded based on [`replaysOnErrorSampleRate`](/platforms/javascript/session-replay/#sampling).
- Manually calling the [`replay.start()`](/platforms/javascript/session-replay/understanding-sessions/#manually-starting-replay) method.

The end of a session replay recording can be triggered by:

- User inactivity within the tab or page that’s being recorded. (It's considered inactivity when a user doesn’t click or navigate around the site for more than 15 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)

> - User closes the tab or page that's being recorded.

Reviewer 3: ```suggestion
- User closing the tab or page that's being recorded.
```
